src/content/meta/glossary.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Glossary
+description: A page with the term definitions Giant Swarm uses in the documentation.
+search: false
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - "What is a cloud-native developer platform?"
+  - "What is a developer environment?"
+  - "What is a management cluster?"
+  - "What is a managed app?"
+  - "What is the platform API?"
+  - "What is a platform team?"
+  - "What is smart platform engineering?"
+  - "What is a workload cluster?"
+---
+
+<!-- All headings must be sorted alphabetically -->
+
+## Cloud native developer platform
+
+A cloud-native developer platform is an integrated collection of capabilities exposed through intuitive interfaces that facilitate the application development lifecycle. It offers flexible configuration to build a system that encodes your company practices. It integrates with different open source solutions to enable automation, connectivity, observability, and security for your workloads. You may be familiar with the term ["internal developer platform"](https://internaldeveloperplatform.org/) which is fairly similar but in our case we emphasize the betting for cloud-native technologies to accomplish the desired goals.
+
+## Developer environment
+
+In our context, a developer environment is the entity that defines the space where the developers will deploy their applications. It can be mapped to a Kubernetes namespace(s) or a cluster. The Platform capabilities like access management, security or observability are related to those environments.
```

Comment:
```suggestion
In our context, a developer environment is the entity that defines the space where developers will deploy their applications. It can be mapped to a Kubernetes namespace or cluster. Platform capabilities like access management, security, or observability are related to those environments.
```